storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3114,8 +3114,16 @@ class Rdb_snapshot_status : public Rdb_tx_list_walker {
 
     /* extract table name and index names using the index id */
     std::string table_name = ddl_manager.safe_get_table_name(gl_index_id);
+    if (table_name == "") {
```

Comment:
In the case where the table name could not be retrieved, maybe you can print out the gl_index_id instead?

Also, can use table_name.empty() instead of comparing it with "".